### PR TITLE
manage installation runtime environments

### PR DESCRIPTION
**Pull request by KPostOffice**
## Related Issues and Dependencies

Depends on: thoth-station/storages#2384

also depends on the creation of two new functions in storages `get_kebechet_github_installations_all(...)` and `delete_kebechet_github_app_installation(...)`. PR for these changes will follow once thoth-station/storages#2384 has been merged.

## This Pull Request implements
This will update DB contents to match latest environment before running Kebechet

### CODE REVIEW

File: update_keb_installation.py [hunk 1]
```diff
@@ -59,60 +60,79 @@ def update_keb_installation():
...
+                slug=_SLUG,
+                runtime_environment_name=runtime_environment.name,
+                installation_id=str(project.github_repo.id),
+                requirements=project.pipfile.to_dict(),
+                requirements_lock=project.pipfile_lock.to_dict(),
+                thoth_config=thoth_config,
+            )
+
+        present_installations = db.get_kebechet_github_app_installations_all(slug=_SLUG)
+        cur_env_names = set([r.name for r in to_update])
```
Comment (fridex) [inline update_keb_installation.py:131]:
```suggestion
        cur_env_names = {r.name for r in to_update}
```

File: update_keb_installation.py [hunk 2]
```diff
@@ -59,60 +60,79 @@ def update_keb_installation():
...
+        all_env_names = set([installation["runtime_environment_name"] for installation in present_installations])
```
Comment (fridex) [inline update_keb_installation.py:132]:
```suggestion
        all_env_names = {installation["runtime_environment_name"] for installation in present_installations}
```

**Review by fridex — APPROVED** [review]
looks good to me, thanks! 👍🏻

**Review by harshad16 — APPROVED** [review]
thanks :100:

**Pull request merged**